index.html

```diff
@@ -159,58 +159,77 @@ <h3>Leverage the power of open source in the field</h3>
         </div>
-        <div class="col-lg-4">
-          <div class="feature">
-            <h2>Be anywhere</h2>
-            <img src="images/point_sync.svg" alt="Be anywhere">
-            <p>Edit your data on the go. Seamless integration with <a href="https://qfield.cloud">QFieldCloud</a> allows
-              your team to work on your projects anywhere anytime.</p>
-          </div>
+      </div>
+      <div class="col-lg-4">
+        <div class="feature">
+          <img src="images/icons/files.png" alt="Powerful Forms" style="width: 22%;">
+          <h3>Powerful Forms</h3>
+          <p style="text-align: center">Create rich survey forms with constraints, logic, defaults, and validations — all in QGIS.</p>
         </div>
-        <div class="col-lg-4">
-          <div class="feature">
-            <h2>Open anything</h2>
-            <img src="images/point_formats.svg" alt="Open anything">
-            <p>Open a wide range of spatial data formats, connect to industry-leading spatial databases and consume
-              standardised geowebservices.</p>
-          </div>
+      </div>
+      <div class="col-lg-4">
+        <div class="feature">
+          <img src="images/icons/camera1.svg" alt="Capture Everything" style="width: 25%; margin: 14px;">
+          <h3>Capture everything</h3>
+          <p style="text-align: center">Photos, videos, audio, QR code scanning, and NFC — fully integrated.</p>
         </div>
       </div>
     </div>
+    <div class="row">
+      <!-- Second row of features -->
+      <div class="col-lg-4">
+        <div class="feature">
+          <img src="images/icons/synchronize.png" alt="Seamless Sync & Format Support" style="width: 25%">
+          <h3>Seamless sync<br> & format support</h3>
+          <p style="text-align: center">Sync projects and data in real time with QFieldCloud and work with GeoPackages, KML, GPX, georeferenced PDFs, and more.</p>
+        </div>
+      </div>
+      <div class="col-lg-4">
+        <div class="feature">
+          <img src="images/icons/path.svg" alt="Editing & Processing Tools" style="width: 25%; margin-bottom: 11px;">
+          <h3>Editing <br> & processing tools</h3>
+          <p style="text-align: center">Snap, reshape, freehand draw, edit vertices, and run field-based geometry operations.</p>
+        </div>
+      </div>
+      <div class="col-lg-4">
+        <div class="feature">
+          <img src="images/icons/plugin1.svg" alt="Plugin Support" style="width: 25%; margin-bottom: 3px;">
+          <h3>Plugin<br> support</h3>
+          <p style="text-align: center">Extend QField with custom QML/JavaScript<br>tools and workflows.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+  <section class="section-spacing text-center" style="margin-bottom: 70px;">
+    <h2>Be smart. Think spatial. Work with QField.</h2>
   </section>
 
   <!-- Section #4: customers -->
```

Comment:
The icons are bigger (I like it), just noting that it means there's a reflow to new line here:

![image](https://github.com/user-attachments/assets/368afaaa-3143-4991-a4eb-650867b3d438)